Jenkinsfile

```diff
@@ -201,7 +201,8 @@ pipeline {
     environment {
         FISSILE_DOCKER_REGISTRY = "${params.FISSILE_DOCKER_REGISTRY}"
         FISSILE_DOCKER_ORGANIZATION = "${params.FISSILE_DOCKER_ORGANIZATION}"
-        USE_SLES_STEMCELL = "${params.USE_SLES_STEMCELL}"
+        USE_SLES_STEMCELL = "${params.USE_SLE}"
+        SUSE_STACK = "${params.USE_SLE ? "sle12" : "opensuse42"}"
```

Comment:
Shouldn't this come from `bin/settings/{sle12,opensuse42}.env`?

Reply:
I don't understand the question... this env var is what's selecting between those two files:
https://github.com/SUSE/scf/blob/a07fb19e143696ff0978a730c9fb79b91e48b2cd/make/kube#L10

Do you mean that we'd source those files directly? Some of the values have spaces in them, so they don't work.
https://github.com/SUSE/scf/blob/a07fb19e143696ff0978a730c9fb79b91e48b2cd/bin/settings/sle12.env#L3

Reply:
Ooof. So we end up having to set the same env var (to the same value) in two places. That doesn't sound fun, but I guess is what we have 😢

Reply:
I've added the quotes, but if they show up in the stack description I'll revert it because it will look strange.